saos-import/src/test/java/pl/edu/icm/saos/importer/notapi/constitutionaltribunal/judgment/process/SourceCtJudgmentExtractorTest.java

```diff
@@ -194,56 +192,13 @@ public void extractPublicationDate() {
     @Test
     public void extractJudges() {
         
-        // given
-        
-        String janNowak = "Jan Nowak";
-        String adamKowalski = "Adam Kowalski";
-        String wrongName = "!! 11";
-        
-        SourceJudge sourceScJudge1 = new SourceJudge();
-        sourceScJudge1.setName(janNowak);
-        sourceScJudge1.setSpecialRoles(Lists.newArrayList(JudgeRole.PRESIDING_JUDGE.name(), JudgeRole.REPORTING_JUDGE.name()));
-        
-        SourceJudge sourceScJudge2 = new SourceJudge();
-        sourceScJudge2.setName(adamKowalski);
-        
-        SourceJudge sourceScJudge3 = new SourceJudge();
-        sourceScJudge3.setName(wrongName);
-        
-        
-        SourceJudge sourceScJudgeBlank = new SourceJudge(); // shouldn't be taken into account because it's name is blank
-        
-        
-        sJudgment.setJudges(Lists.newArrayList(sourceScJudge1, sourceScJudge2, sourceScJudge3, sourceScJudgeBlank));
-        
-        
-        Judge judgeJanNowak = new Judge(janNowak, JudgeRole.PRESIDING_JUDGE, JudgeRole.REPORTING_JUDGE);
-        Judge judgeAdamKowalski = new Judge(adamKowalski);
-        when(judgeConverter.convertJudge(janNowak, Lists.newArrayList(JudgeRole.PRESIDING_JUDGE, JudgeRole.REPORTING_JUDGE), correctionList)).thenReturn(judgeJanNowak);
-        when(judgeConverter.convertJudge(adamKowalski, Lists.newArrayList(), correctionList)).thenReturn(judgeAdamKowalski);
-        when(judgeConverter.convertJudge(wrongName, Lists.newArrayList(), correctionList)).thenReturn(null);
-        
-        
-        
-        // execute
+        Judge judge = new Judge("Jan Kowalski");
+        when(sourceJudgeExtractorHelper.extractJudges(sJudgment, correctionList)).thenReturn(Lists.newArrayList(judge));
         
         List<Judge> judges = judgmentExtractor.extractJudges(sJudgment, correctionList);
         
-        
-        
-        // assert
-        
-        assertEquals(2, judges.size());
-        
-        for (Judge judge : judges) {
-            sJudgment.getJudges().contains(judge.getName());
-            if (judge.getName().equals(sourceScJudge1.getName())) {
-                assertThat(judge.getSpecialRoles(), Matchers.containsInAnyOrder(JudgeRole.PRESIDING_JUDGE, JudgeRole.REPORTING_JUDGE));
-            } else {
-                assertThat(judge.getSpecialRoles(), Matchers.containsInAnyOrder(sourceScJudge2.getSpecialRoles().toArray()));
-            }
-        }
-        
+        assertEquals(1, judges.size());
```

Comment:
... and here assert if the returned list is the same object as expected
assertTrue(returnedJudges == judges)